src/app/audiologist-navigation/audiologist-navigation.component.html

```diff
@@ -32,11 +32,12 @@
         <button type="submit"><i class="fa fa-search"></i></button>
       </div>
 
-      <div class="row"><label class="sec-label">TESTS</label></div>
-      <div class="row"><button class="main-btn" (click)="showRecommendedTests()">Recommended</button></div>
-      <!-- <div class="row"><button class="main-btn" (click)="showSuggestedTests()">Suggested</button></div> -->
-      <div class="row"><label class="sec-label">SUMMARY SECTION</label></div>
-      <div class="row"><button class="main-btn" (click)="showSummary()">Summary</button></div>
+    <div class="row"><label class="sec-label">TESTS</label></div>
+    <div class="row"><button class="main-btn" (click)="showRecommendedTests()">Recommended</button></div>
+    <!-- <div class="row"><button class="main-btn" (click)="showSuggestedTests()">Suggested</button></div> -->
+    <div class="row"><label class="sec-label">SUMMARY SECTION</label></div>
+    <div class="row"><button class="main-btn" (click)="showSummary()">Summary</button></div>
+    <div class="row"><button class="main-btn" (click)="submitSurvey()">Submit</button></div>
```

Comment:
Is is possible to place this button at the bottom of the navigation? (Or differentiate it from the other tab buttons?)